requirements.txt

```diff
@@ -1,2 +0,0 @@
-setuptools
```

Comment:
Hey, what's the reason to drop the requirements file?

Reply:
I think it's actually just included with pip

Reply:
Oh, I thought it was pretty standard to ship a requirements.txt file. However, scikit-learn doesn't either, so maybe we also don't need to do so.